security/pkg/nodeagent/caclient/providers/citadel/client.go

```diff
@@ -139,7 +135,11 @@ func (c *citadelClient) getTLSDialOption() (grpc.DialOption, error) {
 				// Load the certificate from disk
 				certificate, err = tls.LoadX509KeyPair(ProvCert+"/cert-chain.pem", ProvCert+"/key.pem")
 				if err != nil {
-					return nil, fmt.Errorf("cannot load key pair: %s", err)
+					// we will return an empty cert so that when user sets the Prov cert path
+					// but not have such cert in the file path we use the token to provide verification
+					// instead of just broken the workflow
+					citadelClientLog.Warnf("cannot load key pair using token instead : %s", err)
```

Comment:
Warnf("cannot load key pair, using token instead : %+v", err)

Reply:
fix it